server/views/pages/explore.njk

```diff
@@ -75,5 +75,9 @@
     </div>
   </div>
 
+  <div data-component-name="Tweet block" data-component-id="explore">
```

Comment:
Ah - got it - Okay, I'll perhaps document this somewhere, as it's a little confusing (for me).